### ISSUE TITLE
steel toe initializer is broken server down

### ISSUE THREAD

**Issue by ridingmower** [SteeltoeOSS/InitializrService#98]
## Initializer does not work when clicking explore
<img width="973" alt="image" src="https://github.com/user-attachments/assets/9f31c103-b384-4394-93c6-f15c621f6faa">

- [ ] adding Dependencies should be in drop down
- [ ] add asp identity users/roles/claims mgmt. dependency

**Comment by TimHess** [comment]
Yeah, it died again several hours after the restart. I restarted it _again_ when I woke up in the morning, shortly after the last comment here. The monitoring has been updated and is pretty quick to alert now. [This](https://github.com/SteeltoeOSS/InitializrService/blob/main/src/InitializrService/Controllers/UiConfigController.cs#L50) is what breaks, and the service as a whole _looks_ available when it breaks, its just that the config object just comes back empty.

I'd like to close this issue since I think the problem is resolved, but I'm not sure what `add asp identity users/roles/claims mgmt. dependency` means - @ridingmower can you elaborate on anything else you're expecting to happen here?
